src/sidebar/components/NotebookView.js

```diff
@@ -148,7 +148,7 @@ function NotebookView({ loadAnnotationsService, streamer }) {
           forcedVisibleCount={forcedVisibleCount}
           isFiltered={hasAppliedFilter}
           isLoading={isLoading}
-          resultCount={resultCount}
+          resultCount={resultCount ?? 0}
```

Comment:
After making the changes in the store module, TS flagged up that `resultCount` may be `null` here, but the `NotebookResultCount` component expects it to be a number. This mismatch was not a problem in practice because when `resultCount` is `null` other properties (eg. `isLoading`) were set in such a way that the value did not get used. This is the kind of assumption that could be broken by some refactoring elsewhere though.